storagenode/pieces/store_test.go

```diff
@@ -237,6 +244,29 @@ func TestTrashAndRestore(t *testing.T) {
 				{
 					pieceID:    testrand.PieceID(),
 					expiration: time.Now().Add(24 * time.Hour),
+					trashDur:   trashDurToBeKept,
+					files: []testfile{
+						{
+							data:      testrand.Bytes(size),
+							formatVer: filestore.FormatV1,
+						},
+					},
+				},
+				{
+					pieceID:    testrand.PieceID(),
+					expiration: time.Now().Add(24 * time.Hour),
+					trashDur:   trashDurToBeKept,
+					files: []testfile{
+						{
+							data:      testrand.Bytes(size),
+							formatVer: filestore.FormatV1,
```

Comment:
how is this piece different from the previous one? should one of them be FormatV0?

Reply:
It's not, I just wanted to prove it works with a bunch of pieces. It has a different pieceID, and different data, so it is a different piece, but it is also a v1 piece.